buildlib/tools/builds.sh

```diff
@@ -377,6 +377,31 @@ build_fuse() {
 	fi
 }
 
+#
+# Build with static library
+#
+build_static() {
+	${WORKSPACE}/contrib/configure-devel --prefix=$ucx_inst
+	$MAKEP
+	$MAKEP install
+
+	cd test/apps/uct_info
+
+	$MAKE
```

Comment:
1. maybe use "$MAKE -C test/apps/uct_info"
2. why is it not compiled by default?

Reply:
> * maybe use "$MAKE -C test/apps/uct_info"

done, but it required to add ```test/apps/uct_info``` path into other commands


> * why is it not compiled by default?

as per offline discussion this makefile requires command line generated using pkg-config, but it is possible only over installed UCX. that is why here is so complex generation of makefile and CI script